test/unittests/pydra/test_interfaces.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from pydra.tasks.nipype1.utils import Nipype1Task
+
+from clinica.pydra.interfaces import (
+    CAPSFileDataGrabber,
+    CAPSGroupDataGrabber,
+    caps_reader,
+)
+from clinica.pydra.query import BIDSQuery, CAPSFileQuery, CAPSGroupQuery
+
+
+def test_bids_reader(tmp_path):
+    from nipype.interfaces.io import BIDSDataGrabber
+
+    from clinica.pydra.interfaces import bids_reader
+
+    task = bids_reader(BIDSQuery({}), tmp_path)
+    assert isinstance(task, Nipype1Task)
+    assert task.name == "bids_reader_task"
+    assert task.inputs.base_dir == tmp_path
+    assert isinstance(task._interface, BIDSDataGrabber)
+
+
+@pytest.mark.parametrize(
+    "query,grabber",
+    [
+        (CAPSFileQuery({}), CAPSFileDataGrabber),
+        (CAPSGroupQuery({}), CAPSGroupDataGrabber),
```

Comment:
Same comment, this construction with an empty dictionary reads weird. What would be the differences between `CAPSFileQuery()` and `CAPSFileQuery({})`.